tweepy/models.py

```diff
@@ -224,7 +224,12 @@ def __ne__(self, other):
             return result
 
         return not result
-
+    
+    def __hash__(self):
+        if hasattr(self, 'id'):
+            return hash(self.id)
```

Comment:
Using `hash` is unnecessary here, since `User.id` is already an integer.
```suggestion
            return self.id
```